main.go

```diff
@@ -81,21 +93,50 @@ func serverFinalizer(server *Server) {
 //
 // FIXME Pass in a list of 8-bit tags defining epochs.
 // The instance will generate its own secret key.
-func NewServer() (*Server, error) {
+func NewServer() (*Server, *string, error) {
 	// FIXME should we runtime.LockOSThread() here?
 	raw := C.randomness_server_create()
 	if raw == nil {
-		return nil, errors.New("failed to create randomness server")
+		return nil, nil, errors.New("failed to create randomness server")
 	}
 	server := &Server{raw: raw}
 	runtime.SetFinalizer(server, serverFinalizer)
-	return server, nil
+
+	var pkOutput [16384]byte
```

Comment:
Please document this is `ppoprf::MAX_SERIALIZED_PK_SIZE`. Or better, re-export it in the generated C header and use the value from there.

Reply:
added a constant